ui-v2/app/templates/dc/kv/-form.hbs

```diff
@@ -11,14 +11,14 @@
         <div>
             <label class="type-toggle">
                 <input type="checkbox" name="json" checked={{if json 'checked' }} onchange={{action 'change'}} />
-                <span>JSON</span>
+                <span>Code</span>
```

Comment:
It's still JSON mode though, right? Would there be a reason to input anything _other_ than JSON? I guess it's really wide open though isn't it.

Reply:
Yup, the stuff in the KV could be YAML, XML, HCL, TOML, even templates and stuff.

Basically the toggle is to add line numbers and colored syntax highlighting, not 'json editing', I think the best single word for that is just 'Code'. Would be good to think through how we approach some sort of linting for all that in the future. Would be nice to do a 'best guess' and async load the linting code, not sure how successful that would be in reality though, might be too much 'automation', so adding some 'Lint <SyntaxHere>' buttons/radios might by the way to go.